src/runtime/runloop.rs

```diff
@@ -63,6 +63,7 @@ where
     /// Poll this runtime without exiting. Discards any value returned from the
     /// root function. The future yields in between revisions and is woken on
     /// state changes.
+    #[deprecated(note = "Blocks indefinitely. Use BindStream instead.")]
```

Comment:
Is it not possible to retrofit this fix onto the RunLoop type?

Reply:
It doesn't make sense to change waker between polls because mpsc receiver doesn't have this ability.

Reply:
I appreciate your attempt to maintain compatibility, but there are very few users of the crate outside of the tree :D. It's fine to change/break the existing RunLoop API as long as the dom examples keep working.

Once you've made the changes and are sure they're breaking, you can run `cargo ofl versions` to bump the minor version of the moxie crate, and the tool will handle updating the other crates in the repo.

Reply:
Another option would be to remove the async integration from RunLoop entirely, although I'd rather avoid adding more types than necessary. If RunLoop and BindStream *can* be the same type and still support the requestAnimationFrame setup, I think they should be.